elk/training/ccs_reporter.py

```diff
@@ -100,14 +100,20 @@ def __init__(
 
         self.norm = None
 
-        self.probe = nn.Sequential(
+        probe_layers: list[nn.Module] = [
             nn.Linear(
                 in_features,
                 1 if cfg.num_layers < 2 else hidden_size,
                 bias=cfg.bias,
                 device=device,
-            ),
-        )
+            )
+        ]
+
+        if self.config.norm == "burns":
+            probe_layers.append(nn.Sigmoid())
```

Comment:
I still don't understand why you're applying a sigmoid only when the normalization is Burns. All the reporters eventually use a sigmoid. The sigmoids are in `losses.py` and `eval.py`. The `credences` passed around are treated as log-odds and we only apply a sigmoid when trying to get probabilities.